assets/eip-7092/IERC7092.sol

```diff
@@ -99,88 +70,54 @@ interface IERC7092 {
     * @param _owner the bondholder address
     * @param _spender the address that has been authorized by the bondholder
     */
-    function approval(address _owner, address _spender) external view returns(uint256);
```

Comment:
@Pandapip1,

We changed the name of `approval` function to `allowance` to keep the same terminology as with the ERC20. Appart from changing the name, nothing has been changed from how this function is implemented.